config/environments/production.php

```diff
@@ -6,11 +6,22 @@
 /** Disable all file modifications including updates and update notifications */
 define('DISALLOW_FILE_MODS', true);
 
-define('S3_UPLOADS_BUCKET', env('S3_UPLOADS_BUCKET'));
-define('S3_UPLOADS_KEY', env('S3_UPLOADS_KEY'));
-define('S3_UPLOADS_SECRET', env('S3_UPLOADS_SECRET'));
-define('S3_UPLOADS_REGION', env('S3_UPLOADS_REGION'));
-define('S3_UPLOADS_ENDPOINT', env('S3_UPLOADS_ENDPOINT'));
+define('S3_UPLOADS_BUCKET', getenv('S3_UPLOADS_BUCKET'));
```

Comment:
`env` is a function from Wordpress. `getenv` is a function provided by PHP langage (builtin). I've seen a lot of people using it on the internet on the `wp-config.php` file with `define()`.

I used it because it's easier to know what exactly the documented PHP function does. Whereas the Wordpress one doesn't seem to be documented. I didn't find much.